vividus-extension-selenium/src/main/java/org/vividus/bdd/steps/ui/validation/DescriptiveSoftAssert.java

```diff
@@ -36,17 +36,17 @@
         if (!isMatches)
         {
             String assertionDescription = getAssertionDescriptionString(actual, matcher);
-            return recordAssertion(format(businessDescription, assertionDescription),
+            return recordAssertionOutcome(format(businessDescription, assertionDescription),
                     format(systemDescription, assertionDescription), isMatches);
         }
         StringDescription description = new StringDescription();
         matcher.describeTo(description);
         String matchedString = description.toString();
-        return recordAssertion(businessDescription + StringUtils.SPACE + matchedString,
+        return recordAssertionOutcome(businessDescription + StringUtils.SPACE + matchedString,
                 systemDescription + StringUtils.SPACE + matchedString, isMatches);
     }
 
-    private boolean recordAssertion(String businessDescription, String systemDescription, boolean passed)
+    private boolean recordAssertionOutcome(String businessDescription, String systemDescription, boolean passed)
```

Comment:
What is the reason to change the name?

Reply:
code smell to have a  method with the same names in parent and in child class